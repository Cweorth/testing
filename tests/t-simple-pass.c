#include <cut.h>

TEST(success) {
    ASSERT(1);
}

