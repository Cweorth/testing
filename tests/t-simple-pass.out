b'[  1] success................................................................\x1b[1;32mOK\x1b[0m\n\nSummary:\n  tests:       1\n  succeeded:   1\n  skipped:     0\n  failed:      0\n'